hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/ChRootedFileSystem.java

```diff
@@ -102,7 +102,7 @@ protected Path fullPath(final Path path) {
    * @param uri base uri
    * @throws IOException 
    */
-  ChRootedFileSystem(final FileSystem fs, URI uri) throws IOException {
+  public ChRootedFileSystem(final FileSystem fs, URI uri) throws IOException {
```

Comment:
Why is this public instead of protected?

Reply:
The same applies to the rest of the private to public changes.

Reply:
Removed this change as its no longer required.